features/media-queries-inverted-colors.yml

```diff
@@ -0,0 +1,6 @@
+name: inverted-colors media feature
+description: The `inverted-colors` CSS media feature sets styles based on whether the user has inverted all colors.
```

Comment:
This description screams for a "Such as when ...", in order to make it easier to understand when/why/how a user might do that. Is this an OS setting similar to high contrast modes?